src/integrations/foursquare/index.js

```diff
@@ -0,0 +1,55 @@
+const axios = require('axios');
```

Comment:
@maxaleks because our architecture is built fully on OOP patterns, so we use classes everywhere